text/XXXX-quality-check-workflow.md

```diff
@@ -0,0 +1,546 @@
+- The **per sequence quality scores** chart, which shows the mean sequence
+  quality scores across the sample (peaks for lower scores in the orange or red
+  areas of the graph are cause for concern).
+- The **per base sequence content**, which shows is there are any biases in what
+  nucleotides are showing up at particular locations in the reads. For whole
+  genome and whole exome data, there should be relatively few biases. However,
+  for RNA-Seq, you will see a bias at the 5' region of the reads.
+- The **per sequence GC content** chart, which is highly informative of
+  contamination in whole genome and whole exome sequencing data. In those two
+  sequencing modalities, you should check to ensure that all of the
+  distributions are tightly following one another—any derivation from the
+  expected distribution is a sign of contamination. Note that sometimes tools
+  like FastQC plot an "expected" distribution of the GC content; this is usually
+  the expectation of WGS data, but not WXS data, which has a different expected
+  distribution. For RNA-Seq, this graph is generally not used except to examine
+  extreme biases at particular positions.
+- The **per base N content** chart, which should generally report very low
+  levels of Ns in modern data.
+- The **sequence length distribution** chart, which we typically ignore in favor
+  of `ngsderive`'s `readlen` plot.
+- The **sequence duplication level** chart, which gives an indication of the
+  percentage of sequences that are duplicated and by what factor (though this is
+  a graph that we find, often, even good data fails FastQC's stringent test).
+- The **overrepresented sequences** chart, which gives an indication if you're
+  sequencing the same sequence over and over.
+- The **adapter content** chart, which we use to ensure there are minimal to no
+  signs of adapters in the data we receive.
+- And finally, the **status checks** view, which gives you an overview of which
+  samples passed/warned/failed different checks. Generally, this is a good
+  overview of the quality of the data, though we find that many of the charts
+  are often marked as "failed" for good data (in particular, the "Per Base
+  Sequence Content" [for RNA-Seq], the "Per Sequence GC Content", and the
+  "Sequence Length Distribution" graphs).
+
+## Specification
+
+These are generic instructions for running each of the tools in our pipeline. We run our pipeline as a [WDL workflow](https://github.com/stjudecloud/workflows/blob/master/workflows/qc/quality-check-standard.wdl). We have supplied examples of the commands used for each package. For the typical memory requirements of each command, please see our [WDL repository](https://github.com/stjudecloud/workflows).
+
+### Dependencies
+
+We presume Anaconda is available and installed. If not, please follow the link to [Anaconda](https://www.anaconda.com/) first.
+
+```bash
+conda create --name bio-qc \
```

Comment:
This is missing `kraken2` and `mosdepth`. It also includes `fastq-screen` which we no longer run.